lighthouse-core/audits/byte-efficiency/uses-request-compression.js

```diff
@@ -48,7 +48,8 @@ class ResponsesAreCompressed extends ByteEfficiencyAudit {
       // we require at least 10% savings off the original size AND at least 1400 bytes
       // if the savings is smaller than either, we don't care
       if (1 - gzipSize / originalSize < IGNORE_THRESHOLD_IN_PERCENT ||
-          gzipSavings < IGNORE_THRESHOLD_IN_BYTES
+          gzipSavings < IGNORE_THRESHOLD_IN_BYTES ||
+          record.transferSize < gzipSize
```

Comment:
Shouldn't this be a `<=` ?
If  `transferSize` is exactly equal to the `gzipSize` then the resource should still be ignored, right?

Reply:
sure, if we want to get nitpicky about it though it should really just be 

`const gzipSavings = Math.min(originalSize - gzipSize, record.transferSize - gzipSize)` 

above but realistically this is just a sanity check that shouldn't be hit given the new mitigation in the gatherer

feel free to open a PR, we're always open to contributions :)